File: tests/test_sum.py
import unittest
from sum import sum

class TestSum(unittest.TestCase):

    def test_sum_zero(self):
        self.assertEqual(sum(0,0), 0)

    def test_sum_three(self):
        self.assertEqual(sum(1,2), 3)

if __name__ == '__main__':
    unittest.main()

File: ruby/tests/test_sum.rb
require 'minitest/autorun'
require_relative '../sum'

class TestSum < Minitest::Test
  def test_sum_zero
    assert_equal 0, sum(0, 0)
  end

  def test_sum_three
    assert_equal 3, sum(1,2)
  end
end
